docs/src/cli/generate-keys.md

```diff
@@ -0,0 +1,88 @@
+# Generate a Keypair and its Public Key
+
+In this section, we'll generate a keypair, query it for its public key,
+and verify you control its private key. Before you begin, you'll need
+to:
+
+* [Install the Solana Tool Suite](../install-solana.md)
+* [Choose a wallet](choose-a-wallet.md)
+
+## Generate an FS Wallet Keypair
+
+Use Solana's command-line tool `solana-keygen` to generate keypair files. For
+example, run the following from a command-line shell:
+
+```bash
+mkdir ~/my-solana-wallet
+solana-keygen new -o ~/my-solana-wallet/my-keypair.json
+```
+
+If you view the file, you will a long list of numbers, such as:
+
+```text
+[42,200,155,187,52,228,32,9,179,129,192,196,149,41,177,47,87,228,5,19,70,82,170,6,142,114,68,85,124,34,165,216,110,186,177,254,198,143,235,59,173,59,17,250,142,32,66,162,130,62,53,252,48,33,148,38,149,17,81,154,95,178,163,164]
+```
+
+This file contains your **unencrypted** keypair. In fact, even if you specify
```

Comment:
Ah! Here it is!  I think this is an important enough fact to be reiterated in the overview